src/ts/utils/prop-functions.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Utility to allow passing a function as a prop from a dash app.
+ * For example label={'function': 'myLabel'}
+ * where 'myLabel' is a function defined in .js file in /assets
+ *
+ * Based on https://github.com/emilhe/dash-extensions-js
+ */
+
+
+function isPlainObject(o) {
+   return (o === null || o === undefined || Array.isArray(o) || typeof o == 'function' || o.constructor === Date ) ?
+           false
+          :(typeof o == 'object');
+}
+
+function isFunction(functionToCheck) {
+   return functionToCheck && {}.toString.call(functionToCheck) === '[object Function]';
+}
+
+
+export function resolveProp(prop, context = {}) {
+    // If it's not an object, just return.
+    if (!isPlainObject(prop)){
+        return prop
+    }
+    // Check if the prop should be resolved a variable.
+    if (prop.function){
+        return resolveVariable(prop, context)
+    }
+    // Check if the prop should be resolved as an arrow function.
+    if (prop.arrow){
+        return (...args) => prop.arrow
+    }
```

Comment:
Seems like this is actually resolving as a constant, isn't it? Should we call this `prop.constant`?

Reply:
Yeah, this is a placeholder for now.  It might be better to just take it out.  (This is all based on dash-extensions.js and it's how dash-leaflet passes functions as props.)